src/config/constants/shared.jsx

```diff
@@ -0,0 +1,74 @@
+const SHARED = [
```

Comment:
There is noo need to store each constant in separate file, we can just define all of them in `index.js`